pages/guides/developing/access-control-vod.en-US.mdx

```diff
@@ -0,0 +1,183 @@
+    }
+  } catch (error) {
+    console.error('Error:', error);
+  }
+};
+
+createWebhook();
+```
+
+You are going to receive back a JSON containing the id of the webhook.
+
+## Step 3: Create an asset with a playback policy webhook
+
+You can now create an asset with a playback policy webhook, passing the id of the webhook you created in step 2.
+
+```tsx
+import { useCreateAsset } from '@livepeer/react';
+
+import { useCallback, useState } from 'react';
+import { useDropzone } from 'react-dropzone';
+
+export const CreateAndViewAsset = () => {
+  const [video, setVideo] = useState<File | undefined>();
+  const {
+    mutate: createAsset,
+    data: asset,
+    status,
+    progress,
+    error,
+  } = useCreateAsset(
+    video
+      ? {
+          sources: [
+            {
+              name: video.name,
+              file: video,
+              playbackPolicy: {
+                type: 'webhook',
+                // This is the id of the webhook you created in step 2
+                webhookId: '<webhook_id>',
+                webhookContext: {
+                  // This is the context you want to pass to your webhook
+                  // It can be anything you want, and it will be passed back to your webhook
+                },
+              },
+            },
+          ] as const,
+        }
+      : null,
+  );
+
+  const onDrop = useCallback(async (acceptedFiles: File[]) => {
+    if (acceptedFiles && acceptedFiles.length > 0 && acceptedFiles?.[0]) {
+      setVideo(acceptedFiles[0]);
+    }
+  }, []);
+
+  const { getRootProps, getInputProps } = useDropzone({
+    accept: {
+      'video/*': ['*.mp4'],
+    },
+    maxFiles: 1,
+    onDrop,
+  });
+
+  const progressFormatted = useMemo(
+    () =>
+      progress?.[0].phase === 'failed'
+        ? 'Failed to process video.'
+        : progress?.[0].phase === 'waiting'
+        ? 'Waiting'
+        : progress?.[0].phase === 'uploading'
+        ? `Uploading: ${Math.round(progress?.[0]?.progress * 100)}%`
+        : progress?.[0].phase === 'processing'
+        ? `Processing: ${Math.round(progress?.[0].progress * 100)}%`
+        : null,
+    [progress],
+  );
+
+  return (
+    <>
+      <div {...getRootProps()}>
+        <input {...getInputProps()} />
+        <p>Drag and drop or browse files</p>
+      </div>
+
+      {createError?.message && <p>{createError.message}</p>}
+
+      {video ? <p>{video.name}</p> : <p>Select a video file to upload.</p>}
+      {progressFormatted && <p>{progressFormatted}</p>}
+
+      <button
+        onClick={() => {
+          createAsset?.();
+        }}
+        disabled={!createAsset || createStatus === 'loading'}
+      >
+        Upload
+      </button>
+    </>
+  );
+};
+```
+
+## Step 4: Configure the player with an access key
+
+When setting up the player, append the access key to the m3u8 URL as a query parameter, like https://playback.livepeer.studio/asset/hls/abcd1234/index.m3u8?accessKey=your-access-key.
```

Comment:
I can update this with the Player props which should be used instead of manually appending `accessKey`